src/sql/workbench/contrib/queryPlan/common/queryPlanInput.ts

```diff
@@ -3,13 +3,13 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { EditorInput, EditorModel, IEditorInput } from 'vs/workbench/common/editor';
```

Comment:
organized imports, the only actual change is removed IFileService and added ITextFileService